code/modules/awaymissions/mission_code/ghost_role_spawners/golems.dm

```diff
@@ -134,6 +134,24 @@
 		user.death()
 		return
 
+/obj/effect/mob_spawn/human/golem/attackby(obj/item/slimepotion/transference/I, mob/user, params)
+	. = ..()
+	if(istype(src, /obj/effect/mob_spawn/human/golem/servant))
+		src.has_owner = FALSE
+	src.flavour_text = null
```

Comment:
`src.` is implied here since the proc is on `/obj/effect/mob_spawn/human/golem`, so that can just be left out.
```suggestion
		has_owner = FALSE
	flavour_text = null
```